### PR TITLE
fix: support jenkins v100 upgrade

### PR DESCRIPTION
**Pull request by likyh**
### Summary
Jenkins v100 needn't scope. So I create a V100 bp with jenkins try to upgrade it and fix some some bugs.

### CODE REVIEW

File: config-ui/src/pages/blueprint/detail/panel/configuration.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ export const Configuration = ({ blueprint, operating, onUpdate, onRefresh }: Pro
             name: plugin.name,
             connectionId: cs.connectionId,
             entities: plugin.entities,
-            selectedEntites: cs.scopes[0].entities,
+            selectedEntites: cs.scopes.length > 0 ? cs.scopes[0].entities : [],
```
Comment (mintsweet) [inline config-ui/src/pages/blueprint/detail/panel/configuration.tsx:67]: `cs.scopes?.[0].entities ?? []`

Comment (likyh) [reply config-ui/src/pages/blueprint/detail/panel/configuration.tsx:67]: kind of difficult to understand.

File: config-ui/src/error/components/bp-upgrade/use-bp-upgrade.ts [hunk 2]
```diff
@@ -118,7 +118,9 @@ export const useBPUpgrade = ({ id, onResetError }: UseBPUpgradeProps) => {
   const upgradeConnection = async (connection: any) => {
     const { plugin, connectionId } = connection;
 
-    const scopeList = await Promise.all(connection.scope.map((sc: any) => upgradeScope(plugin, connectionId, sc)));
+    const scopeList = connection.scope
```
[the remark this answers is not in the document]
Comment (likyh) [reply config-ui/src/error/components/bp-upgrade/use-bp-upgrade.ts]: fixed

File: config-ui/src/pages/blueprint/detail/panel/configuration.tsx (hunk 1 above)
Comment (likyh) [reply config-ui/src/pages/blueprint/detail/panel/configuration.tsx]: fixed

**Review by warren830 — APPROVED** [review]
@mintsweet already approved

**Pull request merged**